### ISSUE TITLE
SSZ ambiguous reference in tests module

### ISSUE THREAD

**Issue by vlopes11** [sigp/lighthouse#385]
## Description

Upon the execution of `cargo +nightly test` in `eth2/utils/ssz` module, we receive

```
error[E0659]: `ssz` is ambiguous (name vs any other name during import resolution)
  --> /home/vlopes/dev/github/lighthouse/eth2/utils/ssz/src/macros.rs:54:9
   |
54 |     use ssz::{Decode, Encode};
   |         ^^^ ambiguous name
   |
```

## Present Behaviour

The impact is the impossibility to execute the tests on a nightly toolchain (and consequentially on a future stable build)

## Expected Behaviour

All tests run should be successful in stable, beta and nightly

## Steps to resolve

There should be a small fix to refer `ssz` module as a member of `self` inside the tests module

**Comment by kirk-baird** [comment]
What version of nightly are you using?

This compiled and ran fine for me on v1.36.0

**Comment by kirk-baird** [comment]
> What version of nightly are you using?
> 
> This compiled and ran fine for me on v1.36.0

Scratch that I can reproduce the error now

**Comment by paulhauner** [comment]
Resolved in #386
